### PR TITLE
Fixes JS error in deployments page

### PR DESCRIPTION
**Pull request by fabianofranz**
Fixes https://github.com/openshift/origin/issues/4597

@liggitt @jwforres

### CODE REVIEW

**Comment by fabianofranz** [comment]
[test]

File: assets/app/scripts/controllers/deployments.js [hunk 1]
```diff
@@ -313,14 +313,16 @@ angular.module('openshiftConsole')
...
-    $scope.deploymentStatus = function(deployment) {
-      return $filter('annotation')(deployment, 'deploymentStatus');
-    };
-
     $scope.deploymentIsInProgress = function(deployment) {
-      return ['New', 'Pending', 'Running'].indexOf($scope.deploymentStatus(deployment)) > -1;
+      return ['New', 'Pending', 'Running'].indexOf(deploymentStatus(deployment)) > -1;
     };
 
+    function deploymentStatus(deployment) {
```
Comment (liggitt) [inline assets/app/scripts/controllers/deployments.js]: comment why this is declared and then assigned

**Comment by liggitt** [comment]
add comment, then lgtm

File: assets/app/scripts/controllers/deployments.js (hunk 1 above)
Comment (fabianofranz) [inline assets/app/scripts/controllers/deployments.js]: done

**Comment by liggitt** [comment]
[merge]

**Pull request merged**